### PR TITLE
New PRS in EMML1832 (no.10)

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created new PRS IDs for personal names that I found in EMML1832. However, I found an existing ID for Yǝʿqabanna, who is documented probably as a pretender to the thrown in the time of Yāgbǝʿ Ṣyon. I did not find any references or relations connected to PRS10327Yeqaba before I rendered it. The record is probably from EAe (that I cannot check at the moment) or from another secondary literature source.

### CODE REVIEW

**Review by eu-genia — APPROVED** [review]

File: new/PRS14309Benyam.xml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-01-25">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">Bǝnyām</persName>
```
Comment (thea-m) [inline new/PRS14309Benyam.xml:48]: fidal missing and to be swapped

File: new/PRS14311AmlakBena.xml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">አምላከ፡ ብነ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">ʾAmlāka Bǝna</persName>
+                    <floruit notBefore="1300" notAfter="1371">14th century</floruit>
+                </person>
+                <listRelation>
+                    <relation name="dc:relation" active="PRS14311AmlakBena" passive="PRS14279AmlakaBena"><desc>Possibly the same
+                        person as <persName ref="PRS14279AmlakaBena"/>, who was mentioned as a deacon in a previous addition.</desc></relation>
```
Comment (thea-m) [inline new/PRS14311AmlakBena.xml:54]: I think that "in a previous addition" might not be a clear formulation to any readers who only see the person record and not the manuscript.

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you, very nice!

**Review by thea-m — APPROVED** [review]

**Comment by eu-genia** [comment]
nb if you notice that the server is down please do not merge, these merges have no chance of making it to the app and have to be repeated, thank you

**Pull request merged**